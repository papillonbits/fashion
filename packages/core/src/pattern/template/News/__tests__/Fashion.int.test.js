import { regular } from './Fashion.int.story'

describe('<Fashion />', () => {
  describe('Render', () => {
    test('must match regular', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
