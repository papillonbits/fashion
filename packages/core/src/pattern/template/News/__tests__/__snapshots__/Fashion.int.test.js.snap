// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Fashion /> Render must match regular 1`] = `
<div>
  <div
    class="fashion"
  >
    <div
      class="fashionHeading"
    >
      <span
        class="heading headingH1"
      >
        Fashion News
      </span>
    </div>
    <div>
      <div
        class="grid"
      />
    </div>
    <button
      class="fashionButton button"
      type="button"
    >
      LOAD MORE
    </button>
  </div>
</div>
`;
